src/globus_sdk/services/flows/client.py

```diff
@@ -312,9 +312,10 @@ def __init__(
             transport_params=transport_params,
         )
         self._flow_id = flow_id
+        user_scope_value = f"flow_{str(flow_id).replace('-', '_')}_user"
         self.scopes = ScopeBuilder(
             resource_server=str(self._flow_id),
-            known_url_scopes=[f"flow_{flow_id}_user"],
+            known_url_scopes=[("user", user_scope_value)],
```

Comment:
I see the purpose for specifying the resource server and user_scope_value, but I don't think I understand why the `"user"` value is now being passed for `known_url_scopes`. Does that have a special meaning?

Reply:
Explained to me in chat, how "user" here is a reference for how to refer to the scope later, for instance `self.scopes.user`. Thanks for explaining @derek-globus!

Reply:
Yep, specifically because `self.scopes.user` is easier than `self.scopes.flow_8705ad07_e5a0_4645_b2c7_80167bf84656_user`